android/app/src/main/res/navigation/navigation.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
this file has all the navigation actions, navigation directions are generated from the actions in this file